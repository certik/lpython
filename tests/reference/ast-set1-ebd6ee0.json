{
    "basename": "ast-set1-ebd6ee0",
    "cmd": "lpython --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/set1.py",
    "infile_hash": "8597083fb19e73c8b35ac8f94f68dc7f71cc1ab5cea9f86350ce3ac9",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-set1-ebd6ee0.stdout",
    "stdout_hash": "d977e64622f6e764c3ca72e051c619d77b02a7855d034fce9d3d2c2b",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}