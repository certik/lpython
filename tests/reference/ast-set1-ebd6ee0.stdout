(Module [(FunctionDef test_Set ([] [] [] [] [] [] []) [(AnnAssign (Name a Store) (Subscript (Name set Load) (Name i32 Load) Load) () 1) (Assign [(Name a Store)] (Set [(ConstantInt 1 ()) (ConstantInt 2 ()) (ConstantInt 3 ())]) ()) (Assign [(Name a Store)] (Set [(ConstantInt 2 ()) (ConstantInt 3 ()) (ConstantInt 4 ()) (ConstantInt 5 ()) (ConstantInt 5 ())]) ()) (AnnAssign (Name b Store) (Subscript (Name set Load) (Name str Load) Load) () 1) (Assign [(Name b Store)] (Set [(ConstantStr "a" ()) (ConstantStr "b" ()) (ConstantStr "c" ())]) ())] [] () ())] [])
